scripts/test/e2e.sh

```diff
@@ -17,6 +17,18 @@ ${SCRIPTS_DIR}/e2e.sh "$@"
 # Reload KUBECONFIG in case deploy was triggered
 . "${SCRIPTS_DIR}/lib/kubecfg"
 
+# Check that we're testing the version we expect
+function check_operator_version() {
+    local opv
+    kubectl --context=$cluster logs -n submariner-operator -l name=submariner-operator --tail=-1 | grep "Submariner operator version"
```

Comment:
This line looks extraneous...

Reply:
Yes, it’s to check what happens here on e2e.

Reply:
Sorry not sure I follow....isn't it basically the same as the command run on the next line?

Reply:
That’s right, this line is just there so the log contents show up in the build log. I found it useful when debugging this on my own setup; if this fails to do the right thing on CI, I’d rather have this info too. I’ll remove the line before this is merged.

Reply:
Doesn't hurt to always print it - how about something like:

```bash
vs=$(kubectl --context=$cluster logs -n submariner-operator -l name=submariner-operator --tail=-1 | grep "Submariner operator version")
echo $vs
opv=$(awk '/$vs/ { print $NF }')
```